src/makeCalculator.js

```diff
@@ -4,7 +4,48 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculate = {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
+
+    subtract(number) {
+      this.result -= number;
+    },
+
+    multiply(number) {
+      this.result *= number;
+    },
+
+    divide(number) {
+      if (number === 0) {
+        console.log('Cannot divide by zero');
+        return;
+      }
```

Comment:
While it's good to handle the case of division by zero, using `console.log` for error reporting is not recommended in a library function. It's better to throw an error so that the user of the library can decide how to handle it. Consider throwing an Error with a descriptive message instead.